.gitignore

```diff
@@ -17,6 +17,7 @@ x64/
 [Oo]bj/
 .dotnet/
 .tools/
+.packages/
```

Comment:
Why is this necessary now?

Reply:
On CI Arcade restores packages to this dir to isolate.
Do we want to do that for all legs or just one to save restore time?

Reply:
Switched to using global nuget cache, except for in test-correctness.ps1, where it restores to `.packages` dir.